lib/active_admin/localizers.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_admin/localizers/resource_localizer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.